.github/workflows/ruff.yml

```
@@ -17,4 +17,4 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - run: pip install --user ruff
-    - run: ruff --format=github --ignore=E501,F405 .
+    - run: ruff --format=github --ignore=E501,E741,F401,F405 .
```

Right now, we ignore some of these in some files only, see: https://github.com/WeblateOrg/weblate/blob/d1c255a73f20131578918bd3546b339271d281d6/setup.cfg#L21-L40

---

Cool. You can use https://pypi.org/project/ini2toml to migrate that all to `pyproject.toml` to get rid of `setup.py`.

---

I've just fixed these two in 1739c56206e5780c813e40fb7ef45dca1ebed529 and 919adfe8e2f4ff3f4a01dab40e3dd0430f5efe48:

```suggestion
    - run: ruff --format=github --ignore=E501,F405 .
```